### DISCUSSION TITLE
IsarError: Cannot perform this operation from within an active transaction

### DISCUSSION THREAD

**Discussion by Djihanegh** [isar/isar#771] [Help] [answered]
Hello,
i'am trying to filter and update an element from a collection, but i get this error...

`IsarError: Cannot perform this operation from within an active transaction`

![image](https://user-images.githubusercontent.com/40029149/195572650-10f842d1-260d-4190-a1de-96d40d993b23.png)

What's wrong ?

**Comment by simc** [comment, marked as answer]
You are using a synchronous query in an asynchronous transaction.
